asterisk/agi/src/Agi/Action/FaxReceiveStatusAction.php

```diff
@@ -101,7 +102,8 @@ public function process()
             $this->agi->error("File $faxTIF does not exists.");
             // Mark this fax as error and save
             $faxInDto->setStatus('error');
-            $this->entityPersister->persistDto($faxInDto, $faxIn);
+            $this->entityTools->persistDto($faxInDto, $faxIn, true);
+            $faxInDto = $this->entityTools->entityToDto($faxIn);
```

Comment:
Maybe we should do this automatically within persistDto: Update both, faxInDto and faxIn models.

Reply:
Will be done in a separate PR :)